test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/050_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua

```diff
@@ -0,0 +1,157 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [PTU] [GENIVI] PolicyTableUpdate is valid and brings "certificate"
+-- [PTU] [GENIVI] PolicyTableUpdate has NO "certificate" and "ForceProtectedService"=OFF at .ini file
+-- [PTU] [GENIVI] SDL must start PTU for any app except navi right after app successfully request to start first secure service
+--
+-- Description:
+-- In case SDL starts PolicyTableUpdate in case of no "certificate" at "module_config" section at LocalPT
+-- and PolicyTableUpdate is valid and brings "certificate" at "module_config" section
+-- and app sends StartService (<any_serviceType>, encypted=true) to SDL
+-- SDL must start TLSHandshake sequence (by sending TLS Hello to this mobile app)
+--
+-- 1. Used preconditions:
+-- ForceProtectedService is set to OFF in .ini file
+-- Communication app exists in LP, no certificate in module_config
+-- Register and activate application.
+-- Send StartService(serviceType = 7 (RPC))
+-- -> SDL should trigger PTU: SDL.OnStatusUpdate(UPDATE_NEEDED)
+-- -> SDL should not respond to StartService_request
+-- -> SDL should not process request to HMI
+--
+-- 2. Performed steps
+-- Send correct policy file, certificate exists in module_config
+--
+-- Expected result:
+-- 1. SDL sends SDL.OnStatusUpdate(UP_TO_DATE)
+-- 2. SDL process request to HMI
```

Comment:
@istoimenova, Please update if sending of SetAudioStreamingIndicator is not required